config/test.js

```diff
@@ -2,4 +2,5 @@
 module.exports = {
   apiHost: 'https://localhost',
   allowErrorSimulation: true,
+  enableClientConsole: true,
```

Comment:
> the aim of setting this to false here was to make tests not output a log of logging stuff.

I sure hope not because that would be the most cruel trick ever played on a developer. It's 100% essential to see logging in test output. However, I would love to hide the test output for passing tests. Maybe that's possible. As for reducing noise, you can use `--quiet`.